server/events/command_runner_test.go

```diff
@@ -276,6 +281,31 @@ func TestRunCommentCommand_ForkPRDisabled_SilenceEnabled(t *testing.T) {
 	vcsClient.VerifyWasCalled(Never()).CreateComment(matchers.AnyModelsRepo(), AnyInt(), AnyString(), AnyString())
 }
 
+func TestRunCommentCommand_NoProjects_SilenceEnabled(t *testing.T) {
```

Comment:
Can you add some tests around pr status checks resets? Might be better to split this test into 1 per command.